.github/workflows/ci-go-cover.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020-present Montgomery Edwards⁴⁴⁸ (github.com/x448).
```

Comment:
Please change the copyright year to 2021

Reply:
the author (x448) did the original thing that I then modified in 2020, I am not authoritative in this matter :-)